src/client/src/core/util/validation/rules.ts

```diff
@@ -361,7 +361,7 @@ export const validFileTypes = function (fileTypes: string[]) {
 export const validEmailFormat = function (value: string): boolean | string {
   return (
     !value ||
-    /^[a-zA-Z0-9.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)+$/.test(
+    /^(([^<>()[\]\\.,;:\s@#\-"]+(\.[^<>()[\]\\.,;:\s@#\-"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(
```

Comment:
need to change BE
`src/plugins/orangehrmCorePlugin/Api/V2/Validator/Rules/Email.php`

Reply:
can you change this regex also